src/element/bounds.test.ts

```diff
@@ -41,3 +56,57 @@ describe("getElementAbsoluteCoords", () => {
     expect(y2).toEqual(20);
   });
 });
+
+describe("getElementBounds", () => {
+  it("rectangle", () => {
+    const [x1, y1, x2, y2] = getElementBounds(
+      _ce({ x: 40, y: 30, w: 20, h: 10, a: Math.PI / 4, t: "rectangle" }),
+    );
+    expect(x1).toBeCloseTo(39.39339828220179);
```

Comment:
Why not `.toEqual`?

Reply:
I was a bit concerned if there could be a floating point precision issue? No such experience though.

Reply:
If there is going to be such an issue, better catch it early than ignore it. I would suggest switching to `toEqual`.

Reply:
OK, let's switch.